src/lib.rs

```diff
@@ -463,14 +471,185 @@ impl<'a> From<LanguageTag<String>> for LanguageTag<Cow<'a, str>> {
     }
 }
 
+#[cfg(feature = "serde")]
+impl From<SerdeLanguageTag> for LanguageTag<String> {
+    fn from(slt: SerdeLanguageTag) -> Self {
+        slt.lang_tag
+    }
+}
+
+impl<T> Into<String> for LanguageTag<T> where T: Deref<Target = str> {
```

Comment:
I am not sure we need these impls anyway. We should maybe move them out (to an other PR?).

Reply:
Since we implemented serialize-deserialize, yes we do not need them anymore. Will remove.